dev/com.ibm.ws.classloading.bells/resources/com/ibm/ws/classloading/bells/internal/resources/BellConfigurationMessages.nlsprops

```diff
@@ -79,3 +79,21 @@ bell.error.ctor.useraction=If an exception occurred in the constructor, review a
 bell.no.services.config=CWWKL0058W: Cannot find the {0} service in META-INF/services folder from the {1} library.
 bell.no.services.config.explanation=There is no matching service declared in the META-INF/services folder.
 bell.no.services.config.useraction= No user action is required.
+
+# BETA
+# {0} - library identifier
+bell.spi.visibility.enabled=CWWKL0059I: BELL SPI visibility is enabled for the {0} library. The BELL will reference the {0} library through a unique class loader that can see SPI packages.
+bell.spi.visibility.enabled.explanation=To prevent applications and resources from seeing SPI packages, the BELL will reference the library through a unique class loader that is not shared with applications that reference the same library.
+bell.spi.visibility.enabled.useraction=Disable SPI visibility in the BELL configuration whenever the BELL must reference a library through a class loader that is shared with applications that reference the same library. Otherwise, no action is required.
+
+# BETA
+bell.spi.visibility.disabled.libref.global=CWWKL0060W: BELL SPI visibility is not enabled for the Liberty global shared library.
+bell.spi.visibility.disabled.libref.global.explanation=BELL SPI visibility is not supported for the Liberty global shared library.
+bell.spi.visibility.disabled.libref.global.useraction=Modify the BELL configuration to reference a library other than the Liberty global shared library.
+
+# BETA: NOT IMPLEMENTED
+# {0} - application identifier
+# {1} - library identifier
+#bell.spi.visible.share.common.libref=CWWKL0061W: The {0} application and BELL share a common library reference, but do not reference the {1} library through a common class loader, because the BELL is enabled for SPI visibility.
```

Comment:
```suggestion
#bell.spi.visible.share.common.libref=CWWKL0061W: The {0} application and BELL share a common library reference, but do not reference the {1} library through a common class loader because the BELL is enabled for SPI visibility.
```

Reply:
Done.  Thanks @ramkumar-k-9286.